packages/grid/src/Grid.css

```diff
@@ -55,7 +55,7 @@
   --grid-groupHeader-color-default: var(--uitk-color-grey-200);
   --grid-shadow-color-default: rgba(0, 0, 0, 0.2);
   --grid-columnDropTarget-color-default: var(--uitk-color-blue-500);
-  --grid-cursor-borderColor-default: #00477b;
+  --grid-cursor-borderColor-default: var(--uitk-color-blue-400);
```

Comment:
We should use the palette and not colors directly. Do any of the palette semantics match your needs? otherwise we could look at adding appropriate tokens